packages/sanity/src/core/form/inputs/PortableText/object/Annotation.tsx

```diff
@@ -96,7 +96,11 @@ export function Annotation(props: AnnotationProps) {
     [Markers, markers, renderCustomMarkers, text, validation]
   )
 
-  const presence = memberItem?.node.presence || EMPTY_ARRAY
+  const presence =
+    memberItem?.node.presence && memberItem?.node.presence.length > 0
```

Comment:
🤔 Could this work instead?

```suggestion
    memberItem?.node.presence?.length
```

Reply:
Indeed much cleaner! Thanks, committing a fix.